website/app/components/doc/image-caption/index.hbs

```diff
@@ -0,0 +1,3 @@
+<div class={{this.classNames}} ...attributes>
```

Comment:
See comments below
```suggestion
<div class="doc-image-caption doc-text-body-small" ...attributes>
```

Reply:
Understood will follow this pattern going forward!